src/chains-config/metadata-consts/kusamaConsts.ts

```diff
@@ -12,7 +12,7 @@ export const kusamaDefinitions: MetadataConsts[] = [
 	{
 		runtimeVersions: [
 			2027, 2028, 2029, 2030, 9000, 9010, 9030, 9040, 9050, 9070, 9080, 9090,
-			9100, 9110, 9111,
+			9100, 9110, 9111, 9122,
```

Comment:
9120 is missing?

Included in v0.9.12 AFAIU

Reply:
DQ: How would I go about verifying that this list of versions is correct?

Reply:
> 9120 is missing?

Kusama doesnt have 9120 actually (or atleast via polkadot-js types known).

> DQ: How would I go about verifying that this list of versions is correct?

https://github.com/polkadot-js/api/blob/master/packages/types-known/src/upgrades/kusama.ts

It is updated pretty religiously.